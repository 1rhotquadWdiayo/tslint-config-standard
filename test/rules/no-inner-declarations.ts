const y = 10

if (y === 0) {
  function test () {}
}
