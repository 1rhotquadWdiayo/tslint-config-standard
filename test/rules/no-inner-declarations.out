no-inner-declarations.ts[4, 20]: block is empty
no-inner-declarations.ts[4, 12]: unused variable: 'test'
no-inner-declarations.ts[4, 3]: move function declaration to program root
